### DISCUSSION TITLE
4.3.1

### DISCUSSION THREAD

**Discussion by arsenetar** [hsoft/dupeguru#1018] [Announcements]
This release fixes #1015 which can result in non-duplicate files being marked as duplicate when using the standard content mode scan.

## Changes Since Last Release
**Full Changelog**: https://github.com/arsenetar/dupeguru/compare/4.3.0...4.3.1

### Features
* Update Italian translations

### Bug Fixes
* Fix issue where cache db exceptions could prevent files being hashed (#1015)
* Add extra guard for non-zero length files without digests to prevent false duplicates

## Other Notes
* Highly recommended for anyone using version 4.2.0 or later to update to this version, as the issue has existed since that version.
* Release files hosted here are signed with GPG

## OS packages

* Debian archives for x64 (binary archive only compatible with systems using python 3.7/3.8/3.9/3.10)
* RPM for x64, created from deb (only compatible with systems using python 3.7/3.8/3.9/3.10)
* macOS release is compatible with Intel and M1 Mac, and is signed, however not with a paid developer subscription (#945)
* Windows Packages have both installer (exe) and portable (zip) versions. **NOTE windows versions were built using python 3.8 and are only compatible with Windows 7 or later.**
* Arch Linux package is maintained by @glubsy at https://aur.archlinux.org/packages/dupeguru/
* Ubuntu PPA built by @eugenesan at https://launchpad.net/~dupeguru/+archive/ubuntu/ppa

<hr /><em>This discussion was created from the release <a href='https://github.com/arsenetar/dupeguru/releases/tag/4.3.1'>4.3.1</a>.</em>

**Comment by yurone** [comment]
can't run dupeguru_macOS_Qt_4.3.1. on macosx 10.14
see screenshot 
https://pasteboard.co/cV4FlO4RfGIa.png 

p.s. this solution worked for me https://github.com/arsenetar/dupeguru/issues/945#issuecomment-1179009279

**Comment by jamesbcd** [comment]
Just wanted to say thanks for this app - I've found it really useful on my mac. Love the concept of reference files - exactly what I was looking for.

**Comment by vriadi** [comment]
Amazing tool!!
